Cargo.toml

```diff
@@ -232,6 +234,7 @@ chromiumoxide = { git = "https://github.com/mattsse/chromiumoxide", features = [
 ], default-features = false, rev = "348967500868c2b5dfcb3930fe483b2ff17b35ff" }
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
 cityhasher = { version = "0.1", default-features = false }
+collapse = "0.1.2"
```

Comment:
I commented on the use of this crate in the unit tests for one of the new migrations [here](https://github.com/openobserve/openobserve/pull/5100/files#r1847362010)

> This `collaped_eq!` macro comes from the `collapse` trait and its function is to make it easy to ignore whitespace differences when comparing two strings. This was helpful for these tests, but if we want to be selective about new crate dependencies and we don't want to introduce this new dependency on the `collapse` crate then I could replace `collase_eq!` with `assert_eq!` and some custom logic to ignore the whitespace differences.